hunting-buddy.lic

```diff
@@ -76,6 +78,14 @@ class HuntingBuddy
     bput("stow my #{description}", 'You put')
   end
 
+  def check_prehunt_buffs
+    return unless @settings.waggle_sets['prehunt_buffs']
```

Comment:
We could even get rid of this check if we modify `buff.lic` to handle an empty spell set. What do you think?